snapcraft/ovn.env

```diff
@@ -1,7 +1,10 @@
 # Set of environment variables for OVN commands
 
-# Load runtime environment variables
-. "${SNAP_COMMON}/data/env/ovn.env"
+# Load runtime environment variables if available
+runtime_env="${SNAP_COMMON}/data/env/ovn.env"
+if [ -r "$runtime_env" ]; then
+    . "${SNAP_COMMON}/data/env/ovn.env"
```

Comment:
There is a variable containing this string above, is not using it here intentional?

Reply:
Thank you, I overlooked it